### ISSUE TITLE
Suggestion: Avoid `rebar3` file in repository

### ISSUE THREAD

**Issue by sidkshatriya** [josefs/Gradualizer#365]
I notice that there is `rebar3` escript file introduced in https://github.com/josefs/Gradualizer/commit/1b8c363bbdc2b3ee4dbe11ccc665324cc8eec77b .

This is a ~900K escript file which is essentially compiled beam VM binary code. Unlike source code, it's difficult to inspect it and understand what it contains. 

I am new to Gradualizer and am trying to understand it but was a bit concerned to see this file in the repo. Perhaps my concerns are unwarranted. Is there a way we can avoid packaging this file in the repo? Users would usually have `rebar3` locally or this can be specified as a pre-requisite.

Also if you run `make` locally I understand that the repo packaged `rebar3` escript is _not_ run. Am I correct? Under what situations would this file be used?

Thanks for your time!

**Comment by zuiderkwast** [comment]
I agree, rebar3 should not be in the repo. This was a review mistake from my side. We should delete it.

**Comment by zuiderkwast** [comment]
Rebar is used for the property tests, but we can require the user to have rebar installed for running that test.

**Comment by sidkshatriya** [comment]
I've pushed up a simple PR to remove the rebar3 script.

If we absolutely wanted to package rebar3 with this repo we should use https://github.com/erlang/rebar3/releases to get the escript. That might be more reliable and can be verified later also.

The rebar3 script that is currently packaged in Gradualizer looks like it was version `3.14.4` . Unfortunately the rebar3 3.14.4 packaged in this repo and the official Erlang repo differ (I checked using `diff`) so I'm guessing it is a local/alternative build with a variation in OTP version perhaps...
